src/components/avatar-rectangle/__tests__/__snapshots__/avatar-rectangle.test.tsx.snap

```diff
@@ -0,0 +1,58 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
**todo:** Looks like you haven't rebased and/or haven't rerun the tests to have Vitest generate the snapshot file instead of Jest.

Reply:
yes, I haven't migrate it 🙏 , have planned to do this my afternoon 👍